style.scss

```diff
@@ -105,44 +105,57 @@ header {
 main {
     flex-grow: 1;
 
-    .work {
+    .hero-section {
+        padding: 16px;
         background-image: url("./img/man.jpg");
-        background-size: cover;
         background-position: center;
-        height: 400px;
+        background-size: cover;
+        height: 300px;
 
         @media (min-width: $screen-md) {
-            height: 600px;
+            height: 400px;
+        }
+
+        @media (min-width: $screen-lg) {
+            height: 500px;
         }
     }
 
-    .text-about-us {
+    .motto {
         text-align: center;
-        padding-top: 100px;
-        margin: 0 300px 0 10px;
+        text-shadow:
+            3px 3px 6px $dark,
+            0 0 16px $primary,
+            0 0 3px $primary;
+        color: $light;
+        display: flex;
+        align-items: center;
         font-size: 1rem;
+        max-width: 200px;
 
         @media (min-width: $screen-sm) {
-            margin: 0 500px 0 30px;
             font-size: 1.5rem;
+            max-width: 400px;
+
         }
 
         @media (min-width: $screen-md) {
-            margin: 0 600px 0 50px;
             font-size: 2rem;
+            max-width: 450px;
         }
 
         @media (min-width: $screen-lg) {
-            margin: 0 600px 0 50px;
-            font-size: 2rem;
+            font-size: 2.5rem;
+            max-width: 450px;
+           
```

Comment:
You already set `max-width` with this value for `$screen-md`, so it's not needed here.